### PR TITLE
Configured Jest

### PR DESCRIPTION
**Pull request by Emilia123x**
Please review Jest Configuration :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,5 +18,7 @@
     "npm-run-all": "4.1.5",
     "wait-on": "5.2.1"
   },
-  "dependencies": {}
+  "dependencies": {
+    "jest": "^29.0.3"
```
Comment (im-klau) [inline package.json:22]: This `jest` dependency isn't needed here, you've just added it in the `design-system` directory which is fine. You can simply remove this line:)

File: packages/frontend/app/package.json [hunk 2]
```diff
@@ -19,7 +19,8 @@
     "react": "17.0.0",
     "react-dom": "17.0.0",
     "react-markdown": "^4.2.2",
-    "react-moment": "^0.9.6"
+    "react-moment": "^0.9.6",
+    "jest": "^29.0.3"
```
Comment (im-klau) [inline packages/frontend/app/package.json:23]: These two dependencies won't be used in the `frontend/app` directory. We'll use Cypress for e2e testing there. `Jest` is required only in the `design-system` directory for component testing.

File: packages/design-system/package.json [hunk 3]
```diff
@@ -15,7 +15,9 @@
     "@storybook/testing-library": "^0.0.13",
     "babel-loader": "^8.2.5"
   },
-  "dependencies": {},
+  "dependencies": {
+    "jest": "^29.0.3"
```
Comment (im-klau) [inline packages/design-system/package.json:19]: could you also add the `@testing-library/react` package? 🙏

**Comment by Emilia123x** [comment]
Hey, correct jest cofiguration, please kindly review. After pushing the request I got some confics, but I was not deleting any changes, just accepted all. Hope this is fine.  @im-klau

**Comment by im-klau** [comment]
> Hey, correct jest cofiguration, please kindly review. After pushing the request I got some confics, but I was not deleting any changes, just accepted all. Hope this is fine. @im-klau

when there's a conflict on Github you need to choose which option is correct (there are always 2 different options to choose), it's not about accepting things

but that's fine 😅 I'll fix it and merge it today

**Review by im-klau — APPROVED** [review]

**Pull request merged**